test/docs/OpenJ9TestUserGuide.md

```diff
@@ -47,8 +47,8 @@ tools should be installed on your test machine to run tests.
     ```
     JAVA_BIN=<path to JDK bin directory that you wish to test>
     SPEC=[linux_x86-64|linux_x86-64_cmprssptrs|...] (platform on which to test)
-    JAVA_VERSION=[SE80|SE90|SE100|SE110|Panama|Valhalla] (SE90 default value)
-    JAVA_IMPL=[openj9|ibm|hotspot|sap] (openj9 default value)
+    JDK_VERSION=[8|9|10|11|Panama|Valhalla] (9 default value)
```

Comment:
maybe we should change default to 8. 9 is not supported any more.

Reply:
updated